### PR TITLE
Framework: Use imports for WordPress modules instead of the `wp` global

### PR DESCRIPTION
**Pull request by youknowriad**
part of #1205 

I would appreciate a quick review for this to avoid many conflicts :)

### CODE REVIEW

**Comment by iseulde** [comment]
Love it. Do you think we can remove the wp global from eslint too, for better testing?

**Comment by iseulde** [comment]
If we need it anywhere else for external stuff, we can force `window.wp`.

**Comment by youknowriad** [comment]
@iseulde Not against it, but maybe there're too much `wp` usage left for this. (All the `wp.api`, `wp.media` usages)

**Comment by iseulde** [comment]
Hm, ran it and it's passes?

**Comment by iseulde** [comment]
Aha forgot it inherits from other config. Will remove commit

**Comment by iseulde** [comment]
In that case, can we change that with Webpack? It seems strange to still have both after this PR.

**Comment by iseulde** [comment]
Mostly asking because I'm not sure how we'll prevent it from being introduced again unknowingly.

**Comment by youknowriad** [comment]
> In that case, can we change that with Webpack?

I'm trying but failing to define these modules in webpack. It seems to accept `wp` as an external module but not `wp.media`. Any idea?

**Comment by iseulde** [comment]
Oh, this has been attempted before: #742.

**Comment by iseulde** [comment]
Yeah, `wp.media` itself is a function... :)

**Comment by youknowriad** [comment]
Yeah, I'm thinking we'd need to add a namespace to resolve this `import { media } from '@wordpress';`

Anyway, I'm keen to merge this as a first step and see how to solve the external wp modules issue.

File: blocks/editable/index.js [hunk 1]
```diff
@@ -19,7 +20,7 @@ import './style.scss';
 import FormatToolbar from './format-toolbar';
 import TinyMCE from './tinymce';
 
-function createElement( type, props, ...children ) {
+function createTinymceElement( type, props, ...children ) {
```
Comment (aduth) [inline blocks/editable/index.js]: Capitalization: We should probably prefer `TinyMCE`, `createTinyMCEElement`

**Review by aduth — APPROVED** [review]

**Pull request merged**